resources/compass/templates/agent-configuration-job.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: compass-agent-configuration
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: compass-agent-configuration
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: compass-agent-configuration
+subjects:
+  - kind: ServiceAccount
+    name: compass-agent-configuration
+    namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: compass-agent-configuration
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: [""]
+    resources: [configmaps]
+    verbs: [create, get]
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: compass-agent-configuration
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  backoffLimit: 0
+  template:
+    metadata:
+      name: compass-agent-configuration
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      {{ if .Values.global.isLocalEnv }}
+      hostNetwork: true
+      hostAliases:
+        - ip: {{ .Values.global.minikubeIP }}
+          hostnames:
+            - "{{ .Values.global.gateway.host }}.{{ .Values.global.ingress.domainName }}"
+      {{ end }}
+      shareProcessNamespace: true
```

Comment:
Do you need that?

Reply:
not anymore. Was needed when job has istio sidecar.